entries/QUnit.only.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.only">
+	<title>QUnit.only()</title>
+	<signature>
+		<argument name="name" type="String">
+			<desc>Title of unit being tested</desc>
+		</argument>
+		<argument name="callback">
+			<desc>Function to close over assertions</desc>
+			<type name="Function">
+				<argument name="assert" type="Assert"></argument>
+			</type>
+		</argument>
+	</signature>
+	<desc>
+		Adds a test to be exclusively ran
```

Comment:
How about "Adds a test to exclusively run, preventing all other tests from running."